public/pages/ReviewAndCreate/components/ModelConfigurationFields/ModelConfigurationFields.tsx

```diff
@@ -253,26 +253,45 @@ export const ModelConfigurationFields = (
       !props.validModel &&
       props.validationFeatureResponse.hasOwnProperty('message')
     ) {
-      return (
-        <EuiCallOut
-          title="issues found in the model configuration"
-          color="danger"
-          iconType="alert"
-          size="s"
-          style={{ marginBottom: '10px' }}
-        >
-          {/* Callout can either display feature subissue which related to a specific 
-          feature issue or display a feature_attribute issue that is general like
-          more then x anomaly feature or dulicate feature names */}
-          {props.validationFeatureResponse.hasOwnProperty('sub_issues') ? (
-            getFeatureValidationCallout(props.validationFeatureResponse)
-          ) : (
-            <ul>
-              <li>{JSON.stringify(props.validationFeatureResponse.message)}</li>
-            </ul>
-          )}
-        </EuiCallOut>
-      );
+      if (isEqual(props.validationFeatureResponse.validationType, 'model')) {
```

Comment:
Recommend moving lines 256-294 to a helper file in `ReviewAndCreate` so it can be used by `DetectorDefinitionFields` component as well. Looks like the callout logic is duplicated there, with the only difference being the title of the blocker callout - either `"Issues found in the detector settings"` or `"issues found in the model configuration"`

Reply:
ModelConfigurationFields has additional logic in how it displays the message for any blocker issues. This is due to needing to parse the sub issues for feature attributes issues that are also of type detector/blocker. I could refactor it but it would also mean adding more if checks inside depending if the issue response is feature response from `ModelConfigurationsFields` or detector settings from `DetectorDefinitionFields`. I'll try to see if its less messy to refactor and added the checks or to leave it as is

Reply:
Ok, missed that separate logic. Probably fine to leave separate if there's too much unique logic per section.